### ISSUE TITLE
RemoveX produces error when the related category is empty

### ISSUE THREAD

**Issue by nmquangg** [volatiletech/sqlboiler#954]
### What version of SQLBoiler are you using (`sqlboiler --version`)?
4.5.0

### What is your database and version (eg. Postgresql 10)
Postgresql 12

### If this happened at runtime what code produced the issue? (if not applicable leave blank)
```
func (o *Address) RemoveXs(exec boil.Executor, related ...*X) error {
	var err error
	query := fmt.Sprintf(
		"...",
		strmangle.Placeholders(dialect.UseIndexPlaceholders, len(related), 2, 1),
	)
	values := []interface{}{o.ID}
	for _, rel := range related {
		values = append(values, rel.ID)
	}
	...
}
```

### Further information. What did you do, what did you expect?
`RemoveXs` returns an error when `len(related)` is empty as it attempts to execute an sql with `IN ()`, which is invalid. 
It is expected that when `len(related)` is empty, `RemoveXs` does nothing.

**Comment by aarondl** [comment]
Will fix. Thanks for the report.

**Comment by natsukagami** [comment]
Seems to be fixed by 5983717dbc081418da2e12291ef8d816eb64af47
